wiki/support/modding-with-phone.md

```diff
@@ -0,0 +1,117 @@
+---
+sidebar: auto
+---
+# Quest modding with Android
+## Installing BMBF with an Android Phone
+
+* [Requirements](#requirements)
+* [Enable Developer mode on your Quest](#how-to-enable-developer-mode-on-the-oculus-quest)
+* [Setup your Phone](#setup-your-phone)
+* [Installing BMBF with your Phone](#installing-bmbf-with-your-phone)
+* [Setup Beat Saber](#setup-beat-saber)
+
+#### Requirements
+
+* An Android Phone or Android Tablet (IPhones or IPads are not supported)
+* A **paid** Version of Beat Saber on the Oculus Quest Store
+* A Cable to connect your Quest to your Phone (If your phone charges over USB C the charger that was included with your
+Quest should work)
+
+
+#### How to enable developer mode on the Oculus Quest
+First head to [the oculus dashboard](https://dashboard.oculus.com/) and sign in with your Facebook Account.
+You'll need to provide a credit card to verify.
+If you don't have a credit card you can do the steps [here](#i-have-no-credit-card).
+After you successfully logged in and verified you can create your new Organization.
+
+![DevModeDashboard](https://github.com/ComputerElite/wiki/blob/main/BM/DevModeDashboard.png)
+
+![DevModeCreateOrg](https://github.com/ComputerElite/wiki/blob/main/BM/DevModeCreateOrg.png)
+
+After you created it, you can enable Developer mode in the oculus app on your smartphone.
+
+##### Enable Developer mode in the oculus app
+
+![EnableDevMode](https://github.com/ComputerElite/wiki/blob/main/BM/EnableDevMode.png)
+
+#### I have no credit card
+[Original Source](https://www.reddit.com/r/sidequest/comments/jaxy4u/cant_verify_oculus_developer_account/?utm_source=amp&utm_medium=&utm_content=post_body).
```

Comment:
would be better to just pull the instructions into here

Reply:
not quite sure what you mean. I'm just linking the original source. The lines below show how to do it